cuegui/cuegui/Utils.py

```diff
@@ -247,7 +247,7 @@ def findJob(job):
         return None
     if isStringId(job):
         return opencue.api.getJob(job)
-    if not re.search("^([a-z0-9]+)\-([a-z0-9\.]+)\-", job, re.IGNORECASE):
```

Comment:
Is it necessary to remove the period here? Should we just add the underscore?

Reply:
@kateedge100 Friendly ping

Reply:
Apologies for the delay, yes I believe we can leave the period here and just add the underscore.

Reply:
Can we get a test around this logic?